servers/partner-ci/common.yaml

```diff
@@ -630,7 +634,7 @@
           properties-file: properties_file.txt
       - shell: |
           #!/bin/bash -xeu
-          UBUNTU_MIRROR_URL="${MIRROR_HOST:?}${UBUNTU_MIRROR_ID:?}/"
+          UBUNTU_MIRROR_URL="${MIRROR_HOST:?}/pkgs/snapshots/${UBUNTU_MIRROR_ID:?}/"
```

Comment:
add to the common props the next property: 
**MIRROR_HOST=http://mirror.seed-cz1.fuel-infra.org/pkgs/snapshots/**